corehq/motech/dhis2/static/dhis2/js/dhis2_map_settings.js

```diff
@@ -131,8 +159,8 @@ hqDefine('dhis2/js/dhis2_map_settings', [
         self.submit = function (form) {
             var dataSetMaps = [];
             for (var i = 0; i < self.dataSetMaps().length; i++) {
-                var dataSetMap = self.dataSetMaps()[i];
-                dataSetMaps.push(dataSetMap.serialize());
+                var map = self.dataSetMaps()[i];
+                dataSetMaps.push(map.serialize());
```

Comment:
It took me a while to realize the difference between `self.dataSetMaps` and `dataSetMaps`. What do you think of simplifying this to somethin glike:

`var serializedMaps = _.map(self.dataSetMaps(), function(map) { return map.serialize(); });`

Reply:
Good idea.